src/components/Main.vue

```diff
@@ -64,7 +64,7 @@
 import Blurb from "./Blurb";
 import Summary from "./Summary.vue"
 import uniqueId from 'lodash.uniqueid';
-const URL = "http://localhost:5000";
+const URL = "https://api.biascorrect.org/";
```

Comment:
Note that in usage of this variable, we already supply the trailing comma:

https://github.com/gender-bias/gender-bias-web/blob/1c50aa48d39f7768a8411facbe5872974f92f56f/src/components/Main.vue#L109

In other words, we don't want to include the trailing slash both here _as well as_ in the URL construction code.
```suggestion
const URL = "https://api.biascorrect.org";
```